Flatpak/org.DolphinEmu.dolphin-emu.yml

```diff
@@ -18,7 +18,10 @@ finish-args:
   # required for the emulated bluetooth adapter feature to work.
   - --allow=bluetooth
   - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  # required to disable the screensaver in various desktops
```

Comment:
The portal works without any holes in the sandbox, ideally we should be able to drop all of these.

Is this only meant as a temporary measure for people stuck on old versions?

Reply:
Ideally every desktop has their own portal implementation but they don't. I know MATE [1] does not support it or it's completely broken. And it was broken in plasma <6.1.3 [2]. I haven't used XFCE4 in a while so it might work?

> Is this only meant as a temporary measure for people stuck on old versions?

It's more about actually making it work at all.

[1] https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/UICommon/DBusUtils.cpp#L68
[2] https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/UICommon/DBusUtils.cpp#L110

Reply:
Just tested on xfce4 in a xubuntu vm and it blanks the screen at the set timeout.

Reply:
It's a shame because I tested both kde and gnome and it worked perfectly fine there without any permissions.

But for these other desktops I guess there's no way around it, it's a problem even on more mainstream things like the steam deck because of the ancient kde releases valve ships.

Hopefully we'll be able to drop the permissions in a few years.